### ISSUE TITLE
error happens when wrong password entered

### ISSUE THREAD

**Issue by lordadamson** [csurfer/gitsuggest#14]
```
[adam@macbook ~]$ gitsuggest lordadamson

INFO: Authentication (with password) have higher rate limits.
INFO: Skipping password might cause failure due to rate limit.

Password (to skip press enter):
Generating suggestions...
Traceback (most recent call last):
  File "/usr/bin/gitsuggest", line 9, in <module>
    load_entry_point('gitsuggest==0.0.8', 'console_scripts', 'gitsuggest')()
  File "/usr/lib/python3.5/site-packages/gitsuggest/commandline.py", line 61, in main
    gs = GitSuggest(arguments.username, password)
  File "/usr/lib/python3.5/site-packages/gitsuggest/suggest.py", line 50, in __init__
    self.__populate_repositories_of_interest(username)
  File "/usr/lib/python3.5/site-packages/gitsuggest/suggest.py", line 114, in __populate_repositories_of_interest
    user = self.github.get_user(username)
  File "/usr/lib/python3.5/site-packages/github/MainClass.py", line 167, in get_user
    "/users/" + login
  File "/usr/lib/python3.5/site-packages/github/Requester.py", line 172, in requestJsonAndCheck
    return self.__check(*self.requestJson(verb, url, parameters, headers, input, cnx))
  File "/usr/lib/python3.5/site-packages/github/Requester.py", line 180, in __check
    raise self.__createException(status, responseHeaders, output)
github.GithubException.BadCredentialsException: 401 {'documentation_url': 'https://developer.github.com/v3', 'message': 'Bad credentials'}
```

maybe we could handle the bad credentials exception by just printing out wrong password try again

**Comment by csurfer** [comment]
It definitely will complain about wrong credentials if you enter wrong credentials. As the command says, please enter your username and for password either key in the actual one to authenticate and receive a authenticated handle or without typing anything press enter to let the script know you wish to proceed in a unauthenticated way.

**Comment by lordadamson** [comment]
yes I understand that it complains :D 
I am suggesting a better way of complaining and I have created a pull request for it. kindly take a look at it and let me know what you think.

**Comment by csurfer** [comment]
Fixed in v0.0.9
